src/v2/components/screens/SelectImage/Action/styles.ts

```diff
@@ -43,14 +43,9 @@ export const Image = styled.img`
   border-radius: 4px;
 `;
 
-export const ImageContainer = styled.button<{ selected?: boolean }>`
+export const ImageContainer = styled.div<{ selected?: boolean }>`
```

Comment:
shouldn't it technically be a button since it's clickable?

Reply:
As I understand here not all but only the last image for upload is clickable which is already an `<input type="file"/>` thus button is not needed.

![Screenshot 2022-08-04 at 4 59 30 AM](https://user-images.githubusercontent.com/2182307/182729918-1cd5e005-8244-4989-ada1-e6084c19aa80.png)

Reply:
we should allow them to be selected. they can be like default images for the user to pick from

Reply:
In that case these images should be uploaded to IPFS I think